### PR TITLE
App: Added more strings in resources

### PR DESCRIPTION
**Pull request by afischerdev**
Moved more hardcoded strings to resource.
Initial pull request was #645
Please note: this contains no translations.

### CODE REVIEW

File: brouter-routing-app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -13,4 +13,60 @@
...
+    <string name="msg_alternative">Alternative</string>
+    <string name="msg_status_result">version = BRouter-%1$s \n
+        mem = %2$s \n
+        distance = %3$s km\n
+        filtered ascend = %4$s m\n
+        plain ascend = %5$s m\n
+        estimated time = %6$s</string>
+    <string name="msg_read_wpt_error">Error reading waypoints</string>
+    <string name="msg_no_wpt">coordinate source does not contain any waypoints!</string>
+    <string name="msg_to_much_wpts">"coordinate source contains too much waypoints: %1$d (please use from/to/via names)</string>
```
Comment (devemux86) [inline brouter-routing-app/src/main/res/values-ar/strings.xml:70]: The `"` in `"coordinate`should not be needed?

Comment (afischerdev) [reply brouter-routing-app/src/main/res/values-ar/strings.xml]: You are right

**Comment by devemux86** [comment]
@afischerdev very nice, there are a lot of strings!

Please use `String.format(...)` with the positional arguments as seen [here](https://github.com/abrensch/brouter/pull/645/files#diff-03852fb54ad6211a87ff9ab58249df163027c7ba3e4d0d7acc2d98ffa9eb140d).

**Comment by devemux86** [comment]
@afischerdev 

Although the other method is Android specific, Android shows warnings when we build a release apk.
So I had to use in the other pull request the known Java method which passes all tests without issues.

**Comment by afischerdev** [comment]
@devemux86 
Ok, I'll have an eye on that.

**Comment by devemux86** [comment]
@afischerdev there are more texts we need to move to `strings.xml`:
- [BInstallerActivity.java#L282](https://github.com/abrensch/brouter/blob/master/brouter-routing-app/src/main/java/btools/routingapp/BInstallerActivity.java#L282)

- All dialogs in [BInstallerActivity.java#L344](https://github.com/abrensch/brouter/blob/master/brouter-routing-app/src/main/java/btools/routingapp/BInstallerActivity.java#L344) titles / message / buttons:
  - `DIALOG_CONFIRM_DELETE_ID`
  - `DIALOG_CONFIRM_NEXTSTEPS_ID`
  - `DIALOG_CONFIRM_GETDIFFS_ID`
  - `DIALOG_NEW_APP_NEEDED_ID`

File: brouter-routing-app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -13,4 +13,83 @@
...
+    <string name="no">No</string>
+    <string name="yes">Yes</string>
+
+    <string name="title_delete">Confirm Delete</string>
+    <string name="summary_delete">Really delete?</string>
+    <string name="title_version">Version Problem</string>
+    <string name="summary_version">The base version for tiles has changed. What to do?</string>
+    <string name="action_version1">Continue with current download, delete other old data</string>
+    <string name="action_version2">Select all for download and start</string>
+    <string name="action_version3">Cancel now, complete on an other day</string>
```
Comment (devemux86) [inline brouter-routing-app/src/main/res/values/strings.xml:81]: Should be "another" day?

File: brouter-routing-app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -13,4 +13,83 @@
...
+    <string name="title_version_diff">Version Differences</string>
+    <string name="summary_version_diff">The base version for some tiles is different. What to do?</string>
+    <string name="action_version_diff1">Download all different tiles</string>
+    <string name="action_version_diff2">Drop all different tiles</string>
+    <string name="action_version_diff3">Cancel now, complete on an other day</string>
```
Comment (devemux86) [inline brouter-routing-app/src/main/res/values/strings.xml:86]: Should be "another" day?

File: brouter-routing-app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -13,4 +13,83 @@
...
+    <string name="summary_new_version">The new data version needs a new app. Please update BRouter first</string>
+
+    <string name="msg_download_failed">Download failed</string>
+    <string name="msg_download_cancel">Download cancelled</string>
+    <string name="msg_download_succeed">Download succeeded</string>
+    <string name="msg_download_started">"Downloading .. "</string>
```
Comment (devemux86) [inline brouter-routing-app/src/main/res/values/strings.xml:92]: The quotes here are not needed?

File: brouter-routing-app/src/main/res/values-el/strings.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
     <plurals name="numberOfSegments" tools:ignore="MissingQuantity">
-        <item quantity="one">%d τμήμα</item>
+        <item quantity="one">%d τμήμα</item>compare cl
```
Comment (devemux86) [inline brouter-routing-app/src/main/res/values-el/strings.xml:4]: The "compare cl" is error here, we must remove it.

**Pull request merged**